README.md

```diff
@@ -7,6 +7,8 @@ SSLContext should be usable as a drop in replacement for applications that are c
 This code was originally based on the Tomcat Native code, however it has been fairly extensively modified to more closely
 align with JSSE and to support dynamic linking.
 
+wildfly-openssl is compatible with both the OpenSSL 3.0 series and the OpenSSL 1.1.1 series.
```

Comment:
Just something very minor, instead of "wildfly-openssl is...", let's use "WildFly OpenSSL is..."